### PR TITLE
Remove the stickhelmet

### PR DESCRIPTION
**Pull request by chochem**
sorry for the nerf.

these stickhelmets:
image
image

### CODE REVIEW

**Review by Alastors — COMMENTED** [review]
How dare you 😭

**Comment by NotAPenguin0** [comment]
A tragic loss

File: src/main/java/mods/natura/items/StickItem.java [hunk 1]
```diff
@@ -26,7 +25,6 @@ public StickItem() {
     @SideOnly(Side.CLIENT)
     @Override
     public IIcon getIconFromDamage(int meta) {
-        if (meta >= textureNames.length) return Items.leather_helmet.getIconFromDamage(meta);
```
Comment (miozune) [inline src/main/java/mods/natura/items/StickItem.java:29]: You can use `Util.getWithFallback` [`a716d2b` (#16)](https://github.com/GTNewHorizons/Natura/pull/16/commits/a716d2b9742040bb0194c152d7135c2c5324eff7)

Comment (chochem) [reply src/main/java/mods/natura/items/StickItem.java:29]: I dont see why I would want to give textures to stuff that doesnt exist. most mods dont.

Comment (chochem) [reply src/main/java/mods/natura/items/StickItem.java:29]: there always is the default fallback: the flame. and it clearly tells you that something is broken and needs fixing.

Comment (miozune) [reply src/main/java/mods/natura/items/StickItem.java:29]: Well in the case I linked it crashed the game. As long as it doesn't, sure

**Review by miozune — APPROVED** [review]

**Pull request merged**